lib/modules/dosomething/dosomething_global/dosomething_global.module

```diff
@@ -38,7 +38,10 @@ function dosomething_global_init() {
 
   // Verify we're dealing with a node edit with no translation specification in the URL
   if ($node_variables['is_node'] && $node_variables['is_node_edit'] && $node_variables['node_edit_language'] == NULL) {
-    dosomething_global_redirect_node_edit($node_variables['node'], $user_variables['user_language']);
+    // Make sure this is a translatable content type
+    if ($node_variables['node']->translations->original != NULL) {
```

Comment:
This could generate php notices or even warnings (depending on how the class structured).
```php
if (empty($node_variables['node']->translations->original)) {
```

Only in very very rare cases you want to compare something with NULL. 99% is either `isset()` or `empty()`.